cluster/store/service.go

```diff
@@ -221,6 +221,9 @@ func (st *Service) Execute(req *cmd.ApplyRequest) error {
 	if st.store.IsLeader() {
 		return st.store.Execute(req)
 	}
+	if cmd.ApplyRequest_Type_name[int32(req.Type.Number())] == "" {
```

Comment:
**suggestion (non-blocking):** is it necessary ? given that we control the client and server ?